modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -1014,7 +1014,8 @@
 		},
 		getDefaultCaptionCss: function(){
 			var style = {};
-			style["display"] = "inline";
+			style["display"] = "inline-block";
+			style["line-height"] = "2";
```

Comment:
@shalommeoded from my test [here](http://www.kaltura.com/index.php/extwidget/preview/partner_id/2290881/uiconf_id/39595521/entry_id/1_epi34r91/embed/dynamic?debugKalturaPlayer) `dispaly:inline-block` is enough. no?

Reply:
@yairans  in some cases, Thai letters can have up to 3-4 additional symbols on top of the character, therefore, I have added a bit more line space to the background.

Reply:
again, as is sow `display:inline-block` solves the issue. did you see else?

Reply:
Yes, in Thai letters, the background will cut in some cases the Thai symbols. If you can see that `display:inline-block` is enough I can alter the fix.

Reply:
This is what I see but better you verify by yourself

Reply:
@yairans 
Just checked, you are right, the difference does not justify to enhance the line-height.